src/js/components/addData/AddDataMeta.jsx

```diff
@@ -118,10 +127,42 @@ export default class AddDataMeta extends React.Component {
         }
     }
 
-    submitMetadata(){
-        this.props.updateMetaData(this.state);
-    }
-
+    submitMetadata(e){
+        let agency = this.state.agency, 
+            endDate = this.state.endDate,
+            dateType = this.state.dateType,
+            year = endDate.substr(3),
+            quarter = endDate.substr(0, 2);  
+        switch (quarter) {
+            case '12': quarter = '3';
```

Comment:
writing cases this way makes it hard to read. would be better to let the case have it's own line with the code below. 
ie.
```
case '12':
        quarter = '3';
        break;
case ...
```